comps/asr/src/integrations/whisper.py

```diff
@@ -57,7 +58,9 @@ async def invoke(
         }
 
         # Send the file and model to the server
-        response = requests.post(f"{self.base_url}/v1/audio/transcriptions", files=files, data=data)
+        response = await asyncio.to_thread(
```

Comment:
Just out of curiosity. Why all others are replacing `requests` with `aiohttp`, except here?

Reply:
Firstly, both aiohttp and asyncio.to_thread can work, I also find they both exist in original codes. Secondly, there exists some API and usage difference between requests.post and aiohttp. Thus in some cases, we just use asyncio.to_thread to avoid too much alignment and debug effort.